terraform/aws/components/mhs/data_remote_state_base.tf

```diff
@@ -0,0 +1,9 @@
+data "terraform_remote_state" "base" {
+  backend = "s3"
+  
+  config = {
+    bucket = var.tf_state_bucket
+    key = "${var.project}-${var.environment}-base.tfstate"
+    region = var.region
+  }
+}
```

Comment:
can you configure your text editor to ensure theres always new line at the end of file?